keyboards/gmmk/pro/rev1/ansi/keymaps/mattgauf/ansi.c

```diff
@@ -136,7 +136,7 @@ led_config_t PROGMEM g_led_config = {{
     4, 2, 2, 4, 4, 2, 2, 4, 2, 2, 4, 4, 2, 2, 4, 4, 2, 2, 4, 4, 4, 4, 4
 }};
 
-const aw20216s_led g_aw20216s_leds[RGB_MATRIX_LED_COUNT] = {
+const aw20216s_led_t g_aw20216s_leds[RGB_MATRIX_LED_COUNT] = {
```

Comment:
If it were me, i would just remove this whole keymap as it has a bunch of other issues. The duplication of a keyboard level file is totally gross, but it wouldn't get compiled in anyway.